infrastructure/k8s/services/hooks.yaml

```
@@ -0,0 +1,48 @@
+project_name: "taskcluster-hooks"
+secrets:
+  azure_account: ".Values.tcAzureAccount"
```

```suggestion
  azure_account: ".Values.tfAzureAccount"
```

this value comes from terraform, thus is has `tf` prepended to it